src/signal_processing_algorithms/energy_statistics.py

```diff
@@ -0,0 +1,309 @@
+"""Energy statistics."""
+from dataclasses import dataclass
+from typing import List, Tuple, Union
+
+import numpy as np
+
+from signal_processing_algorithms.distance import get_distance_matrix
+
+
+@dataclass
+class EnergyStatisticsTestResult:
+    """Class for representing Energy Statistics and permutation test result."""
+
+    e: float
+    t: float
+    h: float
+    e_p: float
```

Comment:
pvalue rather than p would make this clearer for the next three things.

Reply:
Done.